src/muxer.rs

```diff
@@ -466,20 +466,20 @@ fn offset<'a>(original: &(&'a [u8], usize), subslice: &(&'a [u8], usize)) -> usi
     second + subslice.1 - first - original.1
 }
 
-pub fn gen_mkv_prefix<'a>(
-    input: (&'a mut [u8], usize),
-    header: &EBMLHeader,
-    seek_head: &SeekHead,
-    info: &Info,
-    tracks: &Tracks,
-) -> ::std::result::Result<(&'a mut [u8], usize), GenError> {
-    do_gen!(
-        input,
-        gen_call!(gen_ebml_header, header)
-            >> gen_call!(gen_seek_head, seek_head)
-            >> gen_call!(gen_info, info)
-            >> gen_call!(gen_tracks, tracks)
-    )
+#[allow(dead_code)]
+fn gen_mkv_prefix<'a, 'b>(
+    input: (&'b mut [u8], usize),
+    header: &'a EBMLHeader,
+    seek_head: &'a SeekHead,
+    info: &'a Info,
+    tracks: &'a Tracks,
```

Comment:
None of the `'a`s here is needed, you can just omit them all (and keep the `'b`)